### PR TITLE
Initial Property Estimator code import.

### PR DESCRIPTION
**Pull request by SimonBoothroyd**
## Description
A first pass at merging the existing property estimator code into the repo.

### CODE REVIEW

**Comment by j-wags** [comment]
This pull request **introduces 19 alerts** and **fixes 1** when merging 911825fddd49ed3e973d9123472a71e692a1a1c4 into 04a54114193d0137a959adc2829717d301051722 - [view on LGTM.com](https://lgtm.com/projects/g/openforcefield/propertyestimator/rev/pr-67629b4cdd17469e860af0cce672e6c4b1f2fdca)

**new alerts:**

* 7 for Unused import
* 7 for Module is imported with &#39;import&#39; and &#39;import from&#39;
* 1 for Unnecessary pass
* 1 for Unused local variable
* 1 for Except block handles &#39;BaseException&#39;
* 1 for Unreachable code
* 1 for &#39;import \*&#39; may pollute namespace

**fixed alerts:**

* 1 for &#39;import \*&#39; may pollute namespace

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: docs/conf.py [hunk 1]
```diff
@@ -12,14 +12,14 @@
...
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+sys.path.insert(0, os.path.abspath('.'))
 
 
 # -- Project information -----------------------------------------------------
 
-project = 'Property calculation toolkit from the Open Forcefield Consortium.'
+project = 'Property Estimator'
```
Comment (j-wags) [inline docs/conf.py]: That's weird. I've started an issue on the cookiecutter repo to change this. https://github.com/MolSSI/cookiecutter-cms/issues/64

**Comment by SimonBoothroyd** [comment]
> > 1 for 'import *' may pollute namespace
> 
> It would definitely be good to avoid wildcard imports (`import *`) to follow [PEP8](https://www.python.org/dev/peps/pep-0008/).

Agreed - this should be fixed now.

**Comment by j-wags** [comment]
This pull request **introduces 14 alerts** and **fixes 1** when merging 53565d04da8b5c8c643235568a130376433dc606 into 04a54114193d0137a959adc2829717d301051722 - [view on LGTM.com](https://lgtm.com/projects/g/openforcefield/propertyestimator/rev/pr-ecb2aece560a3330fde2d2d42a16156d0b475c07)

**new alerts:**

* 6 for Module is imported with &#39;import&#39; and &#39;import from&#39;
* 4 for Unused import
* 1 for Unnecessary pass
* 1 for Unused local variable
* 1 for Except block handles &#39;BaseException&#39;
* 1 for Unreachable code

**fixed alerts:**

* 1 for &#39;import \*&#39; may pollute namespace

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by j-wags** [comment]
This pull request **introduces 3 alerts** and **fixes 1** when merging 410d307f1e0a7dd65de5a3516e116b1232576dc6 into 04a54114193d0137a959adc2829717d301051722 - [view on LGTM.com](https://lgtm.com/projects/g/openforcefield/propertyestimator/rev/pr-b1c65c1266536d063b5c86956f7fc9582c6f2f3b)

**new alerts:**

* 1 for Unnecessary pass
* 1 for Unused import
* 1 for Unreachable code

**fixed alerts:**

* 1 for &#39;import \*&#39; may pollute namespace

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**